apm.c

```diff
@@ -272,6 +294,9 @@ static PHP_GINIT_FUNCTION(apm)
 	*next = apm_driver_socket_create();
 	next = &(*next)->next;
 #endif
+#ifdef APM_DRIVER_HTTP
+	*next = apm_driver_http_create();
```

Comment:
While the pattern would be to include `next = &(*next)->next;` with the HTTP driver, static analysis doesn't like assignment to an unused variable.